TI-reports/2025/2025-Q2-VD-WG.md

```diff
@@ -0,0 +1,33 @@
+
+## Overview
+**Mission**: The OpenSSF Vulnerability Disclosures Working Group seeks to help improve the overall security of the open source software ecosystem by helping develop and advocate well-managed vulnerability reporting and communication. We serve open source maintainers and developers, assist security researchers, and help downstream open source software consumers.
+
+Last WG TI update was [February 18, 2025](https://github.com/taladrane/tac/blob/main/TI-reports/2025/2025-Q1-Vulnerability-Disclosure-WG.md). TI is progressing as expected. Core attendance and engagement has continued to remain relatively stable. The Linux Foundation Member Summit and VulnCon 2025 have taken place since the last update and included presentations and followup discussion from the WG.
+
+## Vulnerability Dislosures Working Group
+
+### Purpose
+The OpenSSF Vulnerability Disclosures Working Group seeks to help improve the overall security of the open source software ecosystem by helping mature and advocate well-managed vulnerability reporting and communication.
+
+### Current Status
+- Typically have 9-14 attendees for full WG meeting, 3-5 attendees for APAC WG meeting.
+- [Continuing Graduated WG status](https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#to-become-graduated).
+- WG and subprojects participating in, presented at, and subsequently discussed [VulnCon](https://openssf.org/blog/2025/04/14/key-takeaways-from-vulncon-2025-insights-from-the-openssf-community/) in-depth. A vulnerability identification workshop that included purl was held, and the OSV Project shared reflections from the project. VEX was a prominent topic at VulnCon, and it was mentioned that the VEX as an overarching spec is expected to get a new home and the CISA SBOM community was receptive to hosting VEX somewhere neutral.
+- WG leads and a community member presented at the Linux Foundation Member Summit about open source vulnerabilities and how to bridge the gap between discovery and remediation.
+- WG has discussed the CVE funding issues and is exploring additional solutions to better serve the open source community (https://github.com/ossf/wg-vulnerability-disclosures/issues/162, https://github.com/ossf/wg-vulnerability-disclosures/issues/163). We've also discussed the same concerns with the NVD and have discussed potential benefits of maintaining an NVD mirror.
+- WG has discussed a [funding request](https://github.com/ossf/tac/issues/476) for a purl (package URL) proof-of-concept and training for the CVE Program. There was not enough WG support to pass at this time.
+- Sub-project development:
+   - OSV Project > Working on the [OSV schema graduation application](https://github.com/ossf/tac/pull/456).
+   - OpenVEX SIG > Working on a PR to start the donation of [vexflow](https://github.com/carabiner-dev/vexflow), a Go project that handles the lifecycle of VEX information for projects through GitHub issues, to the OpenVEX project. 
```

Comment:
This is weirdly phrased. It reads as if the WG is working on a PR to start the donation when it would actually be the recipient, right? Also, I suggest you use the word "contribution" instead of "donation" to indicate intent of continuing involvement.

Reply:
```suggestion
   - OpenVEX SIG > Working on a PR to start the contribution of [vexflow](https://github.com/carabiner-dev/vexflow), a Go project that handles the lifecycle of VEX information for projects through GitHub issues, to the OpenVEX project. 
```

Reply:
very good point - thank you for calling this out!